src/components/PhotoAlbum/PhotoAlbum.tsx

```diff
@@ -33,10 +33,11 @@ export const PhotoAlbum = ({ photos }: { photos: any[] }) => {
           distance={Math.abs(currentPhoto - index)}
           onClick={() => setCurrentPhoto(index)}
           active={(index === currentPhoto).toString()}
-          src={`https:${photo.fields.file.url}`}
-          height={300}
-          width={600}
+          src={`https:${photo.fields.file.url}?&w=500`}
+          height={280}
+          width={500}
```

Comment:
```suggestion
          height={272}
          width={416}
```

Reply:
i think this would be more coherent with the image's display size.

Reply:
i noticed in the Lighthouse measurements that the parameters in this format were causing a slight "cumulative layout shift", so i choose to set **height={448} width={624}** as i think it aligns better with the image size, do you agree?